android/app/build.gradle

```diff
@@ -24,6 +24,7 @@ if (flutterVersionName == null) {
 apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
+apply plugin: 'com.google.gms.google-services'
```

Comment:
Could we put at this at the bottom (as per https://pub.dev/packages/firebase_auth)?  I don't know if there's any technical reason but it's nice to stay consistent even just for the sake of code being where people are likely to go looking.

Reply:
It should make no difference on a technical level. I'll change it to be at the end of the file.